modules/social_features/social_landing_page/css/social_landing_page.featured.scss

```diff
@@ -51,7 +51,7 @@
     margin-top: 30px;
     margin-bottom: 0.75rem;
   }
-  .card__actionbar {
+  footer.card__actionbar {
```

Comment:
Hey Serghiy,

What is the reason that we're adding the `footer` tag here as a specifier?

Can we instead be more specific by targetting a specific place where this padding should be 0 (or remove it altogether?)